tests/framework/extensions/kubeapi/tokens/patchtokens.go

```diff
@@ -0,0 +1,46 @@
+package tokens
+
+import (
+	"context"
+	"fmt"
+	"github.com/rancher/rancher/pkg/api/scheme"
+	v3 "github.com/rancher/rancher/pkg/apis/management.cattle.io/v3"
+	"github.com/rancher/rancher/tests/framework/clients/rancher"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+var TokenGroupVersionResource = schema.GroupVersionResource{
+	Group:    "management.cattle.io",
+	Version:  "v3",
+	Resource: "tokens",
+}
+
```

Comment:
Can you add a comment here, for our go docs? In the format of 
`// PatchToken ...`

Reply:
Sure thing @igomez06! Comment added ✅ ty ty

Reply:
Also issue testing wasn't back-ported, so not needed here.